x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -231,6 +230,16 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
             },
           })
         );
+      } else {
+        toastNotifications.addDanger({
+          title: i18n.translate(
+            'xpack.ml.dataframe.analytics.create.unableToFetchExplainDataMessage',
+            {
+              defaultMessage: 'Something went wrong fetching analysis fields data.',
```

Comment:
Text nit: In other similar toasts we mainly have text similar to this variant: `An error occurred fetching analysis fields data.` - `Something went wrong` might be a bit too generic.

Reply:
That's true - updated in 1b6a7d93fab691c5e952441398286976a73469da 👍